orchestra/actions/util.py

```diff
@@ -12,20 +13,27 @@
 
 
 def run_script(script,
-               quiet=False,
-               environment: OrderedDict = None,
+               environment: [OrderedDict, dict] = None,
                strict_flags=True,
                check_returncode=True,
                cwd=None,
+               loglevel="INFO",
+               capture_output=False,
+               stdout_loglevel=None,
+               stderr_loglevel=None,
                ):
     """Helper for running shell scripts.
     :param script: the script to run
-    :param quiet: if True the output of the command is not shown to the user,
-    but instead captured and accessible from the `stdout` and `stderr` properties of the returned value.
     :param environment: will be exported at the beginning of the script
     :param strict_flags: if True, a prelude is prepended to the script to help catch errors
     :param check_returncode: if True an exception is raised unless the script returns 0
     :param cwd: if not None, the command is executed in the specified path
+    :param loglevel: log debug informations at this level
+    :param capture_output: if True the output of the command is captured
+            and accessible from the `stdout` and `stderr` properties of the returned value.
+            Otherwise the subprocess has direct access to orchestra stdout/stderr FDs.
+    :param stdout_loglevel: if not None and capture_output is True, stdout will be logged with this loglevel
+    :param stderr_loglevel: if not None and capture_output is True, stderr will be logged with this loglevel
```

Comment:
I don't see a real use case for having different loglevels for `stdout` and `stderr`.